gcp/volume.go

```diff
@@ -158,30 +199,59 @@ func (c *Client) getVolumeByID(volume volumeRequest) (volumeResult, error) {
 		log.Print("Failed to unmarshall response from getVolumeByID")
 		return volumeResult{}, err
 	}
+
+	// volumeID is verified by Terraform. If we use ID = <volumeID>:<region>, we need to revert our ID changes
+	if originalID != "" {
+		result.VolumeID = originalID
+	}
```

Comment:
result.VolumeID is returned by the API, which has the <volumeID> format, is it necessary to set it again here?